spanner/spec/spec_helper.rb

```diff
@@ -74,18 +85,18 @@ def cleanup_instance_resources
   def cleanup_database_resources
     return unless @instance
 
-    with_retry do 
+    with_retry do
       @test_database = @instance.database @database_id
       @test_database&.drop
       @test_database = @instance.database @restored_database_id
       @test_database&.drop
     end
   end
 
-  def cleanup_backup_resources
-    return unless @instance
+  def cleanup_backup_resources instance
```

Comment:
Consider setting the default value of `instance = @instance` (as you did for the other helpers below) so that existing usage doesn't have to change.